src/avt_vimba_camera.cpp

```diff
@@ -803,13 +809,17 @@ bool AvtVimbaCamera::createParamFromFeature(const FeaturePtr feature, std::strin
         feature->GetValue(initial_value);
         feature->GetRange(minimum_value, maximum_value);
         feature->GetIncrement(step);
+        
+        initial_value = std::clamp(initial_value, minimum_value, maximum_value);
 
         rcl_interfaces::msg::FloatingPointRange float_range;
         float_range.from_value = minimum_value;
         float_range.to_value = maximum_value;
-        float_range.step = step;
+        // TODO: setting step throws an exception rclcpp::exceptions::InvalidParameterValueException
```

Comment:
Was this throwing an exception in Galactic only? I don't remember seeing it in Foxy, but maybe it never came up.

Reply:
Well, don't know what happened but, now it is working with step...

Reply:
According to this comment: https://github.com/ros2/rcl_interfaces/blob/master/rcl_interfaces/msg/FloatingPointRange.msg#L12
Continuous range should work.
What camera feature triggers this problem? I don't see it, but maybe my camera doesn't have the feature.
Is the `feature->GetIncrement(step)` call returning something weird for `step`? maybe we can check the return value of the function and reset to zero if it fails?
All float parameters seem to be correctly set to 0 step, which doesn't throw any errors for me.

Reply:
Oh, now I understand why you do not have this problem. Some features in USB cameras returns non zero increments (like, 0.001 or 0.1) and that causes this problems. Forcing them to be 0 solved to problem.

Reply:
Hmm, you are right, I can reproduce the issue by manually setting some step other than zero. For example:
```what():  parameter 'feature/AcquisitionFrameRateAbs' could not be set: Parameter {} doesn't comply with floating point range.```

It seems like a bug to me, since all documentation I can find seems to suggest it should work fine.
Anyway, I agree with leaving it with 0 step (continuous for now).